packages/scene-composer/src/interfaces/interfaces.tsx

```diff
@@ -39,7 +39,7 @@ export interface ISceneDocumentSnapshot {
    *
    * @param specVersion - version of the document format
    */
-  serialize(specVersion: string): string;
+  serialize(specVersion: string, ...stringifyArgs): string;
```

Comment:
why do we have those changes for `stringifyArgs`?

Reply:
These are optional arguments, and what it lets me do is save scene files in pretty-print.

```
JSON.stringify(object); // results in the object being serialized, but single line
JSON.stringify(object, null, '\t'); // allows you to pretty print
```

Doesn't change existing behavior, but gives more options for end users.